problems/20.Count Days Spent Together.gs

```diff
@@ -0,0 +1,47 @@
+function countDaysTogether(arriveAlice: string, leaveAlice: string, arriveBob: string, leaveBob: string): number {
+
+    let Ama = parseInt(arriveAlice.substring(0, 2))
```

Comment:
I am guessing:
Ama means => Alice Month Arrive ? 
Ada => Alice Day Arrive ?

But the Aaday, Alday and Adays is very hard to wrap my head around :D

You need to improve this naming, some important rules:
1) First letter of variable should be lower case
2) I do like short names however you have shorten it to much, it's not that clear..

I would restructure this a bit also. 
First of all you should extract all the logic from 3 up to 13 in function. If you notice you are doing the same calculations exactly for alice and for bob. This function should return number[] with days count

Reply:
A veliko jer je pocetno slovo od Alice,nekako mi pravopis ne dozvoli da ga pisem s malo slovo :D